.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Enhancement
```

Comment:
Is there a reason to not call this "Feature request"? Which imho is a well defined, well known term.